### ISSUE TITLE
Autotesting modules

### ISSUE THREAD

**Issue by headswe** [TMF3/TMF#43]
Modules would contain

```
class TMF_AutoTest {
    class Tes1 {
       code = "_value = ((_this select 0) get3DENAttribute 'MyTestingNess'); _value > 0 && _value <= 10";
       message = "Value must be between 1 and 10";
   };
};
```

**Comment by Sniperhid** [comment]
This seems like a much cooler way to do things I like!

**Comment by Sniperhid** [comment]
Addressed in #57
